msg/AttentionRecognitionTopNItem.msg

```diff
@@ -0,0 +1,2 @@
+float64 value
```

Comment:
@ekaszubski We should only change it to "log_likelihood" if the actual number is a log likelihood -- in other words, log_e(likelihood).  If it's just a likelihood, then we should leave it as "likelihood" until the change happens (which might be soon depending on the numbers we get).